definitions/infra-ibmmq_manager/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: INFRA
+type: IBMMQ_MANAGER
+
+synthesis:
+  rules:
+    - identifier: targetName
+      name: targetName
+      conditions:
+        - attribute: metricName
+          prefix: ibmmq_
+      tags:
+        clusterName:
+        targetName:
```

Comment:
this will be the name of the entity why it needs to be a tag also?

Reply:
actually it doesn't make much sense. I just removed it and also from the other existant prometheus base integration powerdns.
This attribute also is added in all the metrics as a common attribute.